templates/rhsm-katello-reconfigure.erb

```diff
@@ -63,6 +63,16 @@ else
   sed -i "s/baseurl.*/&\n\n$full_refresh_config/g" $CFG
 fi
 
+# also add the katello ca cert to the system wide ca cert store
+if [-d $CA_TRUST_ANCHORS]; then
+  cp %%(ca_cert_dir)skatello-server-ca.pem $CA_TRUST_ANCHORS
+  update-ca-trust
```

Comment:
On some distros you need to do `update-ca-trust enable` first.  I think that may include EL6.  Not sure, just something I always run